src/requestLib.js

```diff
@@ -96,7 +108,7 @@ var RequestLib = {
         });
   
         clientRequest.on('end', function () {
-          RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_END, params.request.method + ' ' + params.request.url, false, {
+          RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_END, params.request.method + ' ' + requestOptions.path, false, {
```

Comment:
we are logging just the `path` here...

Reply:
RequestLib does not have access to host and provided by client. So cant log whole url

Reply:
won't it be inconsistent in the logs?

Reply:
fixed

Reply:
can be resolved...